src/form.js

```diff
@@ -154,18 +154,18 @@ define([
     Form.prototype = {
         isValidHashtag: function(tag) {
             tag = this.normalizeHashtag(tag);
-            return this.hashtagDictionary.hasOwnProperty(tag) && _.isString(this.hashtagDictionary[tag]);
+            return this.hashtagDictionary.hasOwnProperty(tag) && !_.isFunction(this.hashtagDictionary[tag]);
```

Comment:
That seems clearer to me. Could have something like `knownHashtags` and `hashtagTransformations` ?